app/src/main/java/io/pslab/activity/MainActivity.java

```diff
@@ -180,7 +180,7 @@ public void run() {
         }
     }
 
-    private Fragment getHomeFragment() throws IOException {
+    private Fragment getHomeFragment() throws IOException{
```

Comment:
@riyasingh1004 Do we need this change?

Reply:
I was asked to include this change by @cweitat

Reply:
He told you to return the spacing i.e. keep the spacing as it is. Don't change the lines not needed in this PR

Reply:
Oh okay will do so. Sorry for misunderstanding.